backend/workflow_manager/endpoint/exceptions.py

```diff
@@ -69,3 +71,43 @@ class BigQueryTableNotFound(APIException):
         "Please enter correct correct bigquery table in the form "
         "{table}.{schema}.{database}."
     )
+
+
+class UnstractDBException(APIException):
+    status_code = 400
```

Comment:
Please review if this code should be 400 for all inherited classes - a few could be 5xx errors since it fails in the query we've written

Reply:
curently 400 seems to be fine as we are showing all the error. It is not internal, So 400 would make more sense

Reply:
I meant that 4xx errors are usually user caused errors that they can do something about - such as correcting an api key or a table name. If the error itself is caused due to our incorrect behaviour it should be 5xx. Anyway, I'll let you be the judge of it since you have most context